pymc3/distributions/continuous.py

```diff
@@ -3786,6 +3765,42 @@ def logp(value, mu, sigma):
         )
 
 
+def _interpolated_argcdf(p, pdf, cdf, x):
+    index = np.searchsorted(cdf, p) - 1
+    slope = (pdf[index + 1] - pdf[index]) / (x[index + 1] - x[index])
```

Comment:
It looks like we cannot directly access the index of `pdf` or `x` like this. (The `index` variable is an `Elemwise` during the initial graph construction and hence this fails.). What class/function will be a good substitute for this?

Reply:
I think the problem is that you are using symbolic operations inside the rng_fn. Everything there should be non-symbolic AFAIK

Reply:
That indeed is the problem. What I wanted to know was what is the non-symbolic substitute for such operation? (Maybe `numpy.take` , haven't tried that out yet).

Reply:
I think the symbolic ops used in here are all wrapping standard scipy functions

Reply:
I tried fiddling around with `SplineWrapper` but I don't think that'll be of much use here since we are using different methods of `InterpolatedUnivariateSpline` like the `.integral()` and the `.antiderivate()` ( But can we actually use `SplineWrapper` like that ? )

Instead as you mentioned in https://github.com/pymc-devs/pymc3/pull/4746#discussion_r646490859 I just kept the non-symbolic operations in `dist` and forwarded `cdf_points` as an extra argument.